src/FixerFactory.php

```diff
@@ -35,6 +36,10 @@
  */
 final class FixerFactory
 {
+    const HIDDEN_FIXERS = [
```

Comment:
pls change target to 3.1 line, let's keep effort for 2.x only for bugfixes and we can avoid maintenance effort to synchronize all the changes (3.x is 7.1+ so it will cause some conflicts/missing typehintings if we merge to 2.x)

then, we can skip the idea of `hidden fixers`

Reply:
Done.